numba/tests/test_np_functions.py

```diff
@@ -14,12 +14,19 @@
 
 enable_pyobj_flags = Flags()
 enable_pyobj_flags.set("enable_pyobject")
-
 no_pyobj_flags = Flags()
+no_pyobj_flags.set("nrt")
+
+no_pyobj_flags = [no_pyobj_flags]
+enable_pyobj_flags = [enable_pyobj_flags]
+all_flags = enable_pyobj_flags + no_pyobj_flags
```

Comment:
The functional change made is to add the `nrt` flag which is needed to check that the code works outside of object mode, essentially this fixes a bug but I guess its not obvious from the change as it ought to have been present in the first place.